packages/react-server/core/ReactServerAgent/Request.js

```diff
@@ -22,7 +22,8 @@ function Request(method, urlPath, cache) {
 	this._headers = {};
 	this._timeout = null;
 	this._type = "json"; // superagent's default, only for POST/PUT/PATCH methods
-
+	// CORS off by default
+	this._withCredentials = false;
```

Comment:
Oh right. I didn't notice we were modeling this after superagent. That's unfortunate.